### PR TITLE
add postman tests

### LINKED ISSUE #77: Add rpc call test suite for simple testing
### Problem

Currently most endpoint testing is done manually through `curl` or metamask.
We need a quick way to run a collection of calls against a hosted instance - local or deployed

### Solution

Utilize postman (newman CLI) flow to run multiple queries against a hosted relay instance.
- Add transaction submission logic using js-sdk

See [Mirror Node Web3 acceptance test](https://github.com/hashgraph/hedera-mirror-node/tree/main/docs/web3#acceptance-tests) as inspiration.

For a postman.json file see the [Mirror Node web3 postman.json](https://github.com/hashgraph/hedera-mirror-node/blob/main/hedera-mirror-web3/postman.json) for inspiration.

### Alternatives

Add a ts test suite with a CLI option to run

### PR DESCRIPTION
**Pull request by mmrt**
Closes #77 

Adds postman tests mirroring existing tests.

Signed-off-by: Mitchell Martin <mitch@swirldslabs.com>

### CODE REVIEW

File: packages/server/tests/postman.json [hunk 1]
```diff
@@ -0,0 +1,888 @@
+{
+	"info": {
+		"_postman_id": "267baaca-a637-4399-bbc5-7c04e892a4fa",
+		"name": "Relay",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "21244258"
+	},
+	"item": [
+		{
+			"name": "eth_accounts",
```
[the remark this answers is not in the document]
Comment (Nana-EC) [reply packages/server/tests/postman.json:10]: e.g. https://github.com/hashgraph/hedera-mirror-node/blob/main/hedera-mirror-web3/postman.json#L54

[the remark this answers is not in the document]
Comment (Nana-EC) [reply packages/server/tests/postman.json:10]: Actually you got some unsupported. So just an invalid method test case

File: packages/server/tests/postman.json [hunk 2]
```diff
@@ -0,0 +1,888 @@
...
+						"35",
+						"80"
+					],
+					"port": "7546"
+				}
+			},
+			"response": []
+		}
+	]
+}
```
[the remark this answers is not in the document]
Comment (mmrt) [reply packages/server/tests/postman.json]: done

File: packages/server/tests/postman.json [hunk 3]
```diff
@@ -0,0 +1,888 @@
...
+					"mode": "raw",
+					"raw": "{\n    \"id\": \"test_id\",\n    \"jsonrpc\": \"2.0\",\n    \"method\": \"eth_accounts\",\n    \"params\": []\n}",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "http://34.122.35.80:7546",
```
[the remark this answers is not in the document]
Comment (mmrt) [reply packages/server/tests/postman.json]: done

File: README.md [hunk 4]
```diff
@@ -42,6 +42,15 @@ run configuration. You should be able to just run that configuration, and it sho
 
 The relay has a suite of acceptance tests that may be run to confirm E2E operation of the relay in either a `hedera-local-node` or deployed env.
 
+### Postman
+
+First ensure newman is installed locally using `npm`, then execute `newman`.
+
+```shell
+npm install -g newman
+newman run packages/server/tests/postman.json --env-var baseUrl=http://34.122.35.80:7546
```
Comment (Nana-EC) [inline README.md:51]:
```suggestion
newman run packages/server/tests/postman.json --env-var baseUrl=http://localhost:7546
```
or
```suggestion
newman run packages/server/tests/postman.json --env-var baseUrl=http://127.0.0.1:7546
```

File: packages/server/tests/postman.json [hunk 5]
```diff
@@ -0,0 +1,820 @@
...
+		"exec": [
+		  ""
+		]
+	  }
+	}
+  ],
+  "variable": [
+	{
+	  "key": "base_url",
+	  "value": "34.122.35.80:7546",
```
Comment (Nana-EC) [inline packages/server/tests/postman.json:816]:
```suggestion
	  "value": "locahost:7546",
```
or
```suggestion
	  "value": "127.0.0.1:7546",
```
Either but should be consistent with what you put in README

File: packages/server/tests/postman.json [hunk 6]
```diff
@@ -0,0 +1,820 @@
...
+			"type": "text/javascript"
+		  }
+		}
+	  ],
+	  "request": {
+		"method": "POST",
+		"header": [],
+		"body": {
+		  "mode": "raw",
+		  "raw": "{\n    \"id\": \"test_id\",\n    \"jsonrpc\": \"2.0\",\n    \"method\": \"web3_clientVersion\",\n    \"params\": []\n}",
```
Comment (Nana-EC) [inline packages/server/tests/postman.json:776]: Still missing a case where the method called is not applicable
In the example postman.json I linked we did this by calling a non existent method e.g.
```
		  "raw": "{\n    \"id\": \"test_id\",\n    \"jsonrpc\": \"2.0\",\n    \"method\": \"unknown\",\n    \"params\": []\n}",
```

**Review by Nana-EC — CHANGES_REQUESTED** [review]
IP updated and missing no matching method test caes

File: packages/server/tests/postman.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (mmrt) [reply packages/server/tests/postman.json:10]: missed this, added invalid as well

**Review by Nana-EC — APPROVED** [review]
LG. Just DCO issue

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**